src/Command/CreateFamilyCommand.php

```diff
@@ -40,14 +45,32 @@ protected function configure()
     {
         $this
             ->setDescription('Create an Asset Family')
-            ->addArgument('assetFamilyCode', InputArgument::REQUIRED, 'The asset family code to create')
+            ->addArgument('asset-family-code', InputArgument::REQUIRED, 'The asset family code to create')
+            ->addOption('reference-type', null, InputOption::VALUE_OPTIONAL,
+                sprintf(
+                    'Enable if reference is localizable or not. Allowed values: %s|%s|%s',
+                    self::LOCALIZABLE,
+                    self::NON_LOCALIZABLE,
+                    self::BOTH
+                ),
+                self::BOTH
+            )
         ;
     }
 
     public function execute(InputInterface $input, OutputInterface $output)
     {
         $this->io = new SymfonyStyle($input, $output);
-        $this->assetFamilyCode = $input->getArgument('assetFamilyCode');
+        $this->assetFamilyCode = $input->getArgument('asset-family-code');
+        $referenceType = $input->getOption('reference-type');
+        if (!in_array($referenceType, [self::LOCALIZABLE, self::NON_LOCALIZABLE, self::BOTH])) {
+            throw new \InvalidArgumentException(sprintf(
```

Comment:
There is a way of stating this with symfony so you don't need to make the check here.

Reply:
Seen IRL, not found :(